src/itinerary/Itinerary.tsx

```diff
@@ -75,7 +76,7 @@ const Itinerary = ({
           <BlankSeparator />
         </Fragment>
       )}
-      <ul className={cc([{ 'kirk-itinerary--small': isSmall }])}>
+      <ul className={cc([{ 'kirk-itinerary--small': small || !withTime }])}>
```

Comment:
Definitely "nit" since I don't have any other suggestion but that'd be cool to find a different name for `kirk-itinerary--small` since it's not just when it's "small" (and well even so, it doesn't make it smaller, more simplier)

Reply:
`kirk-itinerary--no-sidebar`?

Reply:
`kirk-itinerary--noTime`?